sdk/src/main/java/io/dapr/client/DaprClientHttp.java

```diff
@@ -644,6 +644,18 @@ public void close() {
     client.close();
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Mono<Void> shutdown() {
+    String[] pathSegments = new String[]{ DaprHttp.API_VERSION, "shutdown" };
+    return Mono.subscriberContext().flatMap(
+            context -> client.invokeApi(DaprHttp.HttpMethods.GET.name(), pathSegments,
```

Comment:
We will concert this to POST. GET will still be supported for backwards compatibility but POST is the recommended way. Charlie has a PR for that in runtime.